OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -26,10 +26,10 @@ public enum DamageSource { Self, Killer }
 	public class ExplodesInfo : ConditionalTraitInfo, Requires<HealthInfo>
 	{
 		[WeaponReference, FieldLoader.Require, Desc("Default weapon to use for explosion if ammo/payload is loaded.")]
-		public readonly string Weapon = null;
+		public readonly string Weapon = "UnitExplode";
```

Comment:
Do we want to introduce another mod-specific default here, or are we taking UnitExplode as generic enough?